docs/spec/deprecated-schema-v1.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Update deprecated schema v1 images
+description: Update deprecated schema v1 iamges
+keywords: registry, on-prem, images, tags, repository, distribution, api, advanced, manifest
+---
+
+Updating an image manifest from schema 1 to schema 2 is done by downloading an
+image, applying schema v2, and then uploading the new image. However, this will
```

Comment:
I would mention that it can be done by downloading the manifest, converting to schema 2, then uploading. Or simply by doing a `docker pull` then `docker push`.

Reply:
Having a practical example of how to do this would be good yes. Something like;

> The easiest way to upgrade an image from image manifest version 2, schema 1 to image manifest version 2, schema 2 is to `docker pull` the image, and `docker push` the image with a current version of Docker. Doing so will automatically convert the  image to use the latest specification.

We may want to add a note/warning that that doesn't upgrade outdated images (which may container vulnerabilities). Something along the lines of;

> **Note**: converting an image to image manifest version 2, schema 2 converts the manifest
> format, but does not update the contents of the image itself. Images using manifest version 2, schema 1
> are likely created a long time ago and may contain unpatched vulnerabilities.
> We recommend looking for an alternative image or, if this is an image you maintain, rebuilding the image.
> To update an image, and patch vulnerabilities, the image has to be re-built using an up-to-date base
> image. Refer to the [update FROM statement](#update-from-statement) section below for more details.